streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -220,18 +221,6 @@ public TopologyTestDriver(final Topology topology,
         this(topology.internalTopologyBuilder, config, initialWallClockTimeMs);
     }
 
-    /**
-     * Create a new test diver instance.
-     *
-     * @param builder builder for the topology to be tested
-     * @param config the configuration for the topology
-     */
-    TopologyTestDriver(final InternalTopologyBuilder builder,
```

Comment:
Removed this constructor after rebasing, since recent commits made it no longer necessary.